packages/core/src/http-handler.ts

```diff
@@ -60,6 +62,17 @@ export class HttpHandler {
     return requestContext;
   }
 
+  protected _bindGetFromContext(context: Context): void {
+    context.bind('getFromContext').to(
+      (key: string): Promise<BoundValue> => context.get(key));
+  }
+
+  protected _bindBindElement(context: Context): void {
+    context.bind('bindElement').to(
+      (key: string, value: ValueOrPromise<BoundValue>) =>
+        context.bind(key).to(value));
```

Comment:
Again, this does not support `.toDynamicValue` and `.toClass`. Please change `bindElement` to accept the `key` only and return the new (empty) `Binding` instance.